### DISCUSSION TITLE
Termux on OpenAPK

### DISCUSSION THREAD

**Discussion by mobilenetworkltd** [termux/termux-app#3972] [General]
Hey,

I'm running a open source repo for Android apps at https://www.openapk.net and I would like to have Termuc listed in the repo. Please review the listing at https://www.openapk.net/termux/com.termux/

It is hooked to your GitHub repo so all the updates are automatically pushed to OpenAPK. Let me know if its ok !

If you are interested you can link to it and use our download banners ( https://www.openapk.net/images/openapk-badge.png )

**Comment by agnostic-apollo** [comment]
Which apk are you using, the universal apk on our github release is 101MB, different from the 97MB you have uploaded. Even other architectures do not match. Or are you building yourself, as apks of older releases were never published on github releases.

https://github.com/termux/termux-app/releases/tag/v0.118.0

Linking is only done to official sources where we ourselves push our releases and have control/trust, so will not be possible. There are other sites like apkmirror that host our releases too, but aren't official.

**Reply by mobilenetworkltd** [reply]
Hello, I am very sorry for not responding but somehow we missed it. We are distributing the apk files available from your official github page and linking to it.

**Comment by Grimler91** [comment]
Why isn't x86(/i686) among supported arches?

I guess file size is in bits instead of bytes? That would explain why your file is listed as 102.2 MB, while file on github is 107 MB (107 MB*1e6/1024**2 ≈ 102 MiB)?

![image](https://github.com/termux/termux-app/assets/18900601/1ac6815f-6ed3-4662-bb4a-e04e48a73349)

**Reply by mobilenetworkltd** [reply]
You are correct. To be honest I haven't looked at it. I use the GitHub API for updates, so whenever you release a new version, it is automatically updated on OpenAPK.
